<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Research - Your Name</title>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>

<header>
        <div class="header-splash">
            <p><strong>Chris Robinson</strong><br>
            Studying symbioses and adaptations in natural microbial populations</p>
        </div>
        <nav>
            <ul>
                <li><a href="index.html">Home</a></li>
                <li><a href="about.html">About</a></li>
                <li><a href="research.html">The Research</a></li>
                <li><a href="google-scholar.html">Google Scholar</a></li> <li><a href="media.html">Media</a></li>
                <li><a href="cv.html">CV</a></li>
            </ul>
        </nav>
    </header>

  <main>
    <h2>My Work</h2>

    <p>Every organism is caught in a dance. Within animals and plants, myriad species of bacteria cooperate and compete for space and food. Within bacteria, strange biological entities, such as viruses and plasmids, spread rapidly within a landscape of hostile microbial cells and bacterial slime, facilitating their own reproduction and transmission through cooperation or force. Deeper still, genes, such as transposons, percolate through chromosomes where they sometimes help, sometimes harm, and often do nothing at all. Everywhere and anywhere, there is evidence for this wondrous, synchronized dance.</p>

    <p>This dance is the result of interactions between two or more partners, and these interactions can benefit – or harm – one of or more of these partners. When these interactions become stable in evolutionary time, they become symbioses. I am interested in how and why these symbioses form, what forces maintain them over time, and ultimately, how they might change in the context of new partners or conditions.</p>

    <p>As an evolutionary (micro)biologist, I use a variety of approaches – field work, genomics, population genetics, and community ecology – to understand how interactions among bacteria, viruses, and plasmids drive ecological and evolutionary variation. I tease apart these interactions within the microbiome of one of the most charismatic systems ever studied: the European Honey Bee.</p>

    <h3>Some of the things that I’m working on:</h3>

    <h4>Adaptation in natural microbial populations via longitudinal metagenomics</h4>
    <p>This project started with a fairly simple question: How do viruses in the honey bee microbiome change over time? We can point to two kinds of change: 1) ecological change, where some viral population rises and falls in frequency and 2) evolutionary change, where the same viral population acquires some adaptation that allows it to have a higher frequency than before. Teasing these two kinds of change apart is not a trivial task – especially using metagenomics – but we can nonetheless use approaches from both ecology and evolution to understand how viral populations might change.</p>
    <p>I monthly sampled honey bee hives from 2023 and part of 2024 to generate hundreds of metagenomic libraries with which I am now deep in the weeds. Some motivating questions that I’m pursuing:</p>
    <ul>
        <li>How do differences in virus ecology (viruses that are lytic, temperate, or trapped as prophages) affect measures of diversity?</li>
        <li>Which bacterial-viral gene pairs exhibit the strongest evidence for coevolution, and how do these genes (and their protein products) change over time?</li>
        <li>How do rates of ecological and evolutionary diversification change over time? Do they differ between viruses and bacteria?</li>
    </ul>

    <h4>Viruses with broader host ranges exhibit higher genomic variation</h4>
    <p>Microbiomes are tricky things to study because they contain such a varied cast of characters. Current approaches to recover and assign these characters with some confidence is not a trivial task. Viruses and plasmids are particularly challenging, because they possess few genes that are hallmarks of viral or plasmid lifestyle, and these genes tend to be frequently transmitted from one host to another via a process called horizontal gene transfer.</p>
    <p>Something we often want to know is: what microbial hosts (e.g. bacterial species) are these viruses and plasmids found in? This is important for many reasons; an important reason is that plasmids (and sometimes viruses) carry genes that enable bacterial virulence or provide resistance to antibiotics. However, we’re often limited in our ability to assign plasmids and viruses to hosts within the microbiome due to methodological constraints. To address this limitation, I collaborated with <a href="https://phasegenomics.com/" target="_blank">Phase Genomics</a> to generate several HiC-resolved metagenomes of the honey bee worker. HiC is a technology that biochemically fuses DNA in a cell before the cell breaks upon. Our assumption is that we should observe DNA fuse with DNA in the same cell more often than we observe this DNA fuse with DNA in another cell. Because a significant part of the viral and plasmid life cycle requires being inside a bacterial cell, we can use this approach to understand how often we observe some viral or plasmid DNA fusing with the DNA of some bacterial species in a microbiome. If this DNA is fused often, we might infer that this bacteria was a host for that virus or plasmid.</p>
    <p>With this approach, we showed – perhaps surprisingly – that both viruses and plasmids are distributed among a broad suite of bacterial hosts within the honey bee microbiome. We identified nearly identical antibiotic resistance genes on independent plasmids in different honey bee colonies, suggesting that these genes are moving around between colonies. Finally, in what I think is the most exciting bit, we show that viruses that are predicted to associate with more hosts – either the same viral population associating with multiple bacteria or the same viral population found in multiple honey bee colonies – experience higher measures of genomic diversity and targets of diversifying selection. There’s a lot of cool stuff in this work, so check out the <a href="https://scholar.google.com/citations?user=SesI7nYAAAAJ&hl=en&authuser=1" target="_blank">paper(s)</a>!</p>

    <h4>Community ecology and population genomics in natural RNA virus populations</h4>
    <p>RNA viruses are (probably) the most common pathogens of insects. In the honey bee, these viruses cause all sorts of diseases, and can lead to the collapse of an entire hive. Despite our concern about these viruses and our understanding of how they’re transmitted from insects to humans, we know very little about how they evolve in natural populations of insects or how variation in insect behavior might affect the opportunity for these viruses to move from one host to another (e.g. transmission rate).</p>
    <p>I hypothesized that RNA virus populations found in social bees (such as honey bees and some others) would be more similar (similar ecological and genetic diversity) than if you compared RNA virus populations from the same species of solitary bees. My intuition was that social behaviors, such as shared food and nests, would increase transmission rate and more effectively homogenize RNA virus variation.</p>
    <p>A collaborator had previously sampled 6 different bee species of varied sociality and performed metatranscriptomics. I leveraged this dataset to explore the questions above, but didn’t find strong evidence for my hypothesis. However, I did identify 28 new viral species and found that each bee species harbored a distinct RNA viral community. Utilizing some new approaches that I learned from population genetics, I found evidence for isolation-by-distance in 5 metapopulations of the same RNA virus species which were sampled along a transect from Princeton to Maine. This work was published in <a href="https://www.nature.com/articles/s43705-022-00127-1" target="_blank">ISME Communications</a>.</p>

    <h4>Bioinformatics education and mentorship</h4>
    <p>As a scientist, I wholeheartedly believe that the most important contribution I can make is to inspire and mentor other students. Research, academia, graduate school – none of these things are particularly easy, and are made harder without a community with which to draw on for guidance. I’ve worked to address these inequities by serving as an <a href="https://asm.org/" target="_blank">American Society for Microbiology Future Leaders Fellow</a>, as a member of the <a href="https://www.amnat.org/" target="_blank">American Society for Naturalist Graduate Research Council</a>, and as a <a href="https://www.microbiomedata.org/" target="_blank">National Microbiome Data Collaborative Ambassador (and now champion!)</a>. I’ve hosted multiple bioinformatics workshops for hundreds of students in Ukraine, both remote and in person during my Fulbright Fellowship, and continue to look for any opportunity to expand access to underserved and underrepresented scientific communities. During my PhD, I’ve had the absolute pleasure to mentor several excellent undergraduate students and indirectly mentor tens of others. I’ve served two years as a mentor for the <a href="https://holland.biology.indiana.edu/" target="_blank">IU Jim Holland program</a> where I mentor wonderful high school students on the grotesque art of honey bee dissection.</p>
</main>
    <footer>
        <div class="footer-banner">
            <p>Footer Banner Area</p>
        </div>
    </footer>

</body>
</html>
